manifests/a/AntiMicro/AntiMicro/2.24.0/AntiMicro.AntiMicro.locale.en-US.yaml

```diff
@@ -19,5 +19,6 @@ ReleaseNotes: |-
   - Update Chinese and French translation
   - Other minor improvements
 ReleaseNotesUrl: https://github.com/AntiMicro/antimicro/releases/tag/2.24-final
+InstallationNotes: This package is no longer actively maintained. There is new recommended version of this app - AntiMicroX.antimicrox

```

Comment:
If the package is no longer maintained, should it be removed from our repos, to prevent future potential vulnerabilities or other issues?

Reply:
There's plenty of packages in the repo that aren't actively maintained, so I think this is fine